### PR TITLE
Fix some bad German translations

### PR DESCRIPTION
**Pull request by Wuzzy2**
There were some minor translation issues with German.
“cobblestone” should be “Kopfsteinpflaster”. “Bruchstein” is something else.
Screwdriver text had a newline for some reason; this broke the item tooltip.

Everything else are minor grammatical fixes.

### CODE REVIEW

**Comment by Xanthin** [comment]
To be honest, I had my problems with the title of your PR. Sure, you can't know how much time and effort I spend in finding a good fitting word. Almost everything was a conscious decision after research in different sources. So that probably wasn't an intended offence. 
And contribution of course is very welcome. Thank you.

The main controversy seems to be the translation of cobblestone (since the rest of the PR changes typos and, talking about the screwdriver tooltip an improvement in the game I missed over the time). 
I know why I avoided creating an article for it in the official wiki. But now you got me after all.

_Why did I choose "Bruchstein" over "Kopfsteinpflaster"?_

[Here](https://paste.ubuntu.com/22707213/) is the list of possible translations and explanations I made two years ago for this personaly most difficult word in all the time.

^
This was the texture of cobblestone at this time. It had not much similarity to most of what I saw on photos about "Kopfsteinpflaster" (rounded head) or even any "Pflasterstein" (small cube-shaped). But more with what you see about "Bruchstein" (flat, light mortar).

Even I never owned or played it, I also looked up the minecraft wiki, found their translation and after some weeks considered it to be the best fitting for the texture _and_ the way you get this material in the game. It's not crafted like I would expect it from a "Pflaster". It feels like in this sense you would get wooden planks dropped from chopping trees.

The texture changed over time (and changed, and changed ...) and now looks much more rounded at the tops with removed light mortar. Also it might be debatable if a translation is the right place to query the original naming. 

Since this fork seems to step out of privacy I guess

**Long story short**

your proposed changes are expedient, just the PR title could have been a bit different for my taste.

Btw.,

> Screwdriver text had a newline for some reason; this broke the item tooltip.

The reason was, that back in time when I translated it, longer tooltips were broken using small resolutions to play. The text was not completely readable, so I used the new line to fix that for me. 

This behaviour is obviously fixed (probably since a longer time) and I guess you don't like how it is displayed above the hotbar using UI?
screwinui

**Comment by Wuzzy2** [comment]
Well, ok, if you insist on “Bruchstein” then I won't mind.
But the actual problem is that the original English strings are sometimes just bad or weird.
Like the tree trunks are called just “tree”, there are 4 specific types of “tree” (like “**aspen** tree”) but one unspecific one (just “tree”), etc. But that's something to discuss in Minetest Game.

WIth the screwdriver, the tooltip was kinda broken for me, MT was unable to display the text correctly; the tooltip was too small to contain the whole text.
Are newlines even officially supported in the description? :-/
No, Unified Inventory was not the reason, that would obviously something to report to VanessaE instead. ;-)
Oh, not everyone is using development version of Minetest btw. :P

Oh, and the German translation was already very good, I really just touched rather minor things.

**Comment by Xanthin** [comment]
I just took your changes in like you proposed. eb16ecd329c968bc117bb302509f49f5491853d9
Translation is not the place to fix weird feeling origin words, that was the argument. Thank you, it's good to have a final decision now. :)

And thanks for the feedback.

Em, and please don't ask what I did to mess up the whole thing ... I had hard hours to fix this to get your commit clear in. I'm not a professional. Proven again.
